app/api/auth.py

```diff
@@ -363,3 +365,37 @@ def decorated(*args, **kwargs):
 def environment_details():
     envdump = EnvironmentDump(include_config=False)
     return envdump.dump_environment()
+
+
+@ticket_blueprint.route('/orders/resend-email', methods=['POST'])
+@limiter.limit(
+    '3/hour', key_func=lambda: request.json['data']['order'], error_message='Limit for this action exceeded'
+)
+@limiter.limit(
+    '1/minute', key_func=get_remote_address, error_message='Limit for this action exceeded'
```

Comment:
IMO 1/minute is very less for this threshold. An organizer can go on clicking the resent invite button on the client side so we should increase this. @iamareebjamal what can be a suitable threshold for this endpoint?

Reply:
Maybe `3/minute` ? @shreyanshdwivedi @iamareebjamal

Reply:
60/minute for IP
5/minute for user_id